### DISCUSSION TITLE
欢迎来到灵墨社区！

### DISCUSSION THREAD

**Discussion by elysia-best** [LingmoOS/Community#1] [社区通知]
🎉 欢迎来到灵墨社区！ 🎉

亲爱的开发者们、技术爱好者们，以及所有对编程、设计和创新感兴趣的朋友们，你们好！我们非常高兴你们能加入这个充满活力和创造力的灵墨社区（Lingmo Community）。

📚 灵墨社区是什么？

灵墨社区是一个为技术爱好者、开发者、设计师和所有对创新充满热情的人们而建立的交流平台。在这里，我们分享知识、交流经验、探索新技术，并共同推动技术的边界。无论你是初出茅庐的新手，还是久经沙场的老将，灵墨社区都欢迎你的加入！

🌟 为什么选择灵墨社区？

活跃的交流：我们鼓励社区成员之间的交流和合作。你可以在这里提出问题、分享经验、参与讨论，甚至找到你的合作伙伴或团队。
友好的氛围：我们致力于营造一个友好、互助、尊重的社区氛围。在这里，你可以放心地表达你的想法和观点，不用担心被嘲笑或批评。

🎯 如何参与？

关注我们的GitHub仓库：通过关注我们的仓库，你可以随时了解社区的最新动态和更新。
参与讨论：在我们的讨论区（如这个Discussion板块）发表你的观点、提出问题或分享经验。
贡献内容：如果你有好的技术文档、教程或开源项目，欢迎在社区分享，让更多人受益。
参与活动：关注我们的活动页面，参加我们举办的线上/线下活动，与更多志同道合的人交流和学习。

如果您发现有违社区秩序的情况出现，烦请联系 @LingmoOS/sig-for-community-management 进行处理，谢谢您的支持！

最后，再次感谢大家的加入！让我们一起在灵墨社区共同成长，创造更美好的未来！🚀

**Comment by devLythen** [comment]
好耶 XD

**Comment by Aninana** [comment]
来了(doge)

**Comment by N55743** [comment]
hi

________________________________

**Comment by elysia-best** [comment]
系统的安装密码为 live
________________________________________
***@***.*** ***@***.***> on behalf of N55743 ***@***.***>
Sent: Monday, July 29, 2024 11:12 AM
To: LingmoOS/Community ***@***.***>
Cc: Yinan Qin ***@***.***>; Author ***@***.***>
Subject: Re: [LingmoOS/Community] 欢迎来到灵墨社区！ (Discussion #1)
 
请问密码是什么？
image.png (view on web)
初次安装系统时他要求我输密码，但我并不知到密码是什么。

**Comment by N55743** [comment]
谢谢

________________________________

**Comment by elysia-best** [comment]
可以的，这个在live-build仓库来修改。之前试过一次，似乎会卡在登录，之后看看怎么搞。(●ˇ∀ˇ●)

________________________________________
***@***.*** ***@***.***> on behalf of N55743 ***@***.***>
Sent: Monday, July 29, 2024 11:39 AM
To: LingmoOS/Community ***@***.***>
Cc: Yinan Qin ***@***.***>; Author ***@***.***>
Subject: Re: [LingmoOS/Community] 欢迎来到灵墨社区！ (Discussion #1)
 
提一个建议。像其他发行版那样取消这个安装操作系统时需要输入密码到功能。

________________________________

**Comment by everything411** [comment]


**Reply by elysia-best** [reply]
Strange though. We may remove this from the pre-packed image and provide this as a local source instead. 

CC. @lingmo-dream

**Reply by everything411** [reply]
i start a separate thread on https://github.com/orgs/LingmoOS/discussions/21

**Comment by ColoMovo** [comment]
已经在使用了 awa 很漂亮也很实用 期待后续更新
PS：在安装过程中出现问题：无法正常加载桌面环境，无法调整安装程序窗口，周末附截图（寄宿生，望理解😔）
